### PR TITLE
implement IsInf for opset 10

### PR DESCRIPTION
**Pull request by mindest**
op `IsInf` is available since onnx opset 10.

Add a direct op for `IsInf` and a corresponding test case.

### CODE REVIEW

**Comment by nbcsm** [comment]
https://github.com/onnx/tensorflow-onnx/issues/468

File: tf2onnx/onnx_opset/tensor.py [hunk 1]
```diff
@@ -881,3 +881,10 @@ def version_4(cls, ctx, node, **kwargs):
         reorganize_node = ctx.make_node(node.type, trans1.output, attr={"blocksize": blocksize[0]})
         ctx.make_node("Transpose", reorganize_node.output, {"perm": [1, 2, 3, 0]}, name=node.name, outputs=node.output,
                       shapes=shapes, dtypes=dtypes)
+
+
+@tf_op("IsInf", onnx_op="IsInf")
+class IsInf:
+    @classmethod
+    def version_10(cls, ctx, node, **kwargs):
+        pass
```
Comment (nbcsm) [inline tf2onnx/onnx_opset/tensor.py]: check input dtype in [float, double]

File: tests/test_backend.py [hunk 2]
```diff
@@ -2078,5 +2078,17 @@ def test_space_to_batchnd(self):
         _ = tf.space_to_batch_nd(input_x, block_size, pad, name=_TFOUTPUT)
         self._run_test_case([_OUTPUT], {_INPUT: input_val})
 
+    @check_opset_min_version(10, "is_inf")
+    def test_isinf(self):
+        x_val1 = np.array([1.0, 2.0, -3.0, -4.0], dtype=np.float32).reshape((2, 2))
```
Comment (nbcsm) [inline tests/test_backend.py]: is double type support?
can we try input with a few different ranks, not just (2, 2)

Comment (mindest) [reply tests/test_backend.py]: Yes, double type is also supported.

Comment (nbcsm) [reply tests/test_backend.py]: i mean different rank, e.g. 1d, 2d, 3d tensor.

**Comment by nbcsm** [comment]
@mindest, please also get this resolved.
https://github.com/onnx/tensorflow-onnx/pull/477#discussion_r278816091

**Comment by nbcsm** [comment]
forget to approve before merging...

**Pull request merged**